.github/workflows/publish-release.yml

```diff
@@ -411,7 +411,7 @@ jobs:
       - e2e-admin-tests
       - e2e-upgrade-test
       - check_branch
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
maybe we should upgrade `ghcr.io/goreleaser/goreleaser-cross` while we're at it? Without upgrading this we don't actually increase the minimum glibc version.